controllers.js

```diff
@@ -23,9 +24,19 @@ exports.getArticle = (req, res, next) => {
 exports.getArticleById = (req, res, next) => {
     const article_id = req.params.article_id;
     
-    getArticleByIdModels(article_id)
+    getArticleByIdModel(article_id)
     .then( (article) => {
         res.status(200).send({article});
     })
     .catch(next);
+}
+
+exports.getCommentsById = (req, res, next) => {
+    const article_id = req.params.article_id;
+
+    getCommentsByIdModel(article_id)
+    .then( (articles) => {
+        res.status(200).send(articles);

```

Comment:
when you implement the test for the `404` error of an article_id of `999` or any number that is not found - you will find that it fails because you are not currently checking the database for if the article exists 

Here in the controller you can make use of another model function you have already written - `getArticleById` - and using a `Promise.all` or a promise chain of `.then` it can be used together with your `getCommentsByIdModel`